lib/GLPI/Agent/SNMP/MibSupport/Ubnt.pm

```diff
@@ -10,29 +10,90 @@ use GLPI::Agent::Tools::SNMP;
 
 # See UBNT-MIB
 
-use constant ubnt               => '.1.3.6.1.4.1.41112';
-use constant ubntWlStatApMac    => ubnt . '.1.4.5.1.4.1';
+use constant ubnt              => '.1.3.6.1.4.1.41112';
+use constant ubntWlStatApMac   => ubnt . '.1.4.5.1.4.1';
+
+# See UBNT-UniFi-MIB
+
+use constant unifiVapEssid        => ubnt . '.1.6.1.2.1.6';
+use constant unifiVapName         => ubnt . '.1.6.1.2.1.7';
+use constant unifiApSystemVersion => ubnt . '.1.6.3.6.0';
+use constant unifiApSystemModel	  => ubnt . '.1.6.3.3.0';
+
 
 our $mibSupport = [
     {
         name    => "ubnt",
-        oid     => ubnt
```

Comment:
Here, don't modify the existing detection as you can break existing support.

Better add another detection in the list to have:
```
our $mibSupport = [
    {
        name    => "ubnt",
        oid     => ubnt
    },
    {
        name    => "ubnt-sysobjectid",
        sysobjectid => getRegexpOidMatch(ubnt)
    }
];